src/app/public/modules/flyout/flyout-instance.ts

```diff
@@ -15,19 +15,53 @@ import {
   SkyFlyoutMessageType
 } from './types/flyout-message-type';
 
+/**
+ * The `SkyFlyoutService` launches flyouts and returns a `SkyFlyoutInstance` object to represent
+ * a single displayed flyout.
+ */
 export class SkyFlyoutInstance<T> implements OnDestroy {
+
+  /**
+   * An event that the flyout instance emits when it closes.
+   */
   public closed = new EventEmitter<void>();
+
+  /**
+   * Specifies the instance of the component to display in the flyout.
+   */
   public componentInstance: T;
+
+  /**
+   * Used to communicate with the host component.
+   * @internal
+   */
+  public get hostController(): Subject<SkyFlyoutMessage> {
+    return this._hostController;
+  }
+
+  /**
+   * Indicates if the flyout is open.
+   * @default true
+   */
   public isOpen = true;
 
+  /**
+   * An event that the flyout instance emits when users click the next iterator button.
+   */
   public get iteratorNextButtonClick(): EventEmitter<void> {
     return this._iteratorNextButtonClick;
   }
 
+  /**
+   * An event that the flyout instance emits when users click the previous iterator button.
+   */
   public get iteratorPreviousButtonClick(): EventEmitter<void> {
     return this._iteratorPreviousButtonClick;
   }
 
+  /**
+   * Disables the next iterator button.
```

Comment:
Should there be a default value here? Looks like we missed that in the existing docs, but I assume that if we didn't explicitly mark this as `true` by default, then we should show a default of `false`.

Also, same question as with `isOpen` about whether this property controls the behavior or just reports on it. If the latter, it seems like this should be something more like `Indicates whether the next iterator button is disabled.`

Reply:
Good catch, yes - this is done.